src/api/spec/README.md

```diff
@@ -6,7 +6,13 @@ test things based on the following rules:
 * Every main workflow has a feature test
 
 ## Running the spec
-`bundle exec rake spec`
+```bundle exec rspec```
+
+and to run a single test file:
+
+```
+bundle exec rspec spec/models/user_spec.rb
+```
```

Comment:
Do we need this section?:-) It seems so basic. Any rspec beginners guide might cover this, including the rpsec docu.